File: LibraryAPI/utils/connection_util.py
from psycopg2 import connect
from psycopg2._psycopg import OperationalError
import os


def create_connection():
    try:
        conn = connect(
            host= 'ranieridb.cyysbedq8cqc.us-east-1.rds.amazonaws.com',
            database='postgres',
            user= 'adam',
            password='gatorfan1',
            port='5432'
        )
        return conn
    except OperationalError as e:
        print(e)


connection = create_connection()

File: CommunityLibrariAPI/src/main/java/dev/ranieri/utils/ConnectionUtil.java
package dev.ranieri.utils;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class ConnectionUtil {

    public static Connection createConnection(){
        //  jdbc:postgresql://ranieridb.cyysbedq8cqc.us-east-1.rds.amazonaws.com:5432/postgres?user=adam&password=gatorfan1
        try {
            Connection connection = DriverManager.getConnection("jdbc:postgresql://ranieridb.cyysbedq8cqc.us-east-1.rds.amazonaws.com:5432/postgres?user=adam&password=gatorfan1");
            return connection;
        } catch (SQLException sqlException) {
            sqlException.printStackTrace();
            return null;
        }
    }

    // quick way to test if successful
    public static void main(String[] args) {
        System.out.println(createConnection());
    }
}
